_schemaV2.graphql

```diff
@@ -35,102 +46,52 @@ type AccountRequest {
   notes: String
 }
 
-# Autogenerated input type of AddAssetsToConsignmentSubmission
-input AddAssetsToConsignmentSubmissionInput {
+# Autogenerated input type of AddAssetToConsignmentSubmission
+input AddAssetToConsignmentSubmissionInput {
```

Comment:
The large diff in the `_schema` file worries me a bit. 'Noise' like comments or whitespace, etc is of course fine, but I wonder if there are 'real' changes (here, `Assets` -> `Asset`).

I don't think this file is actually used by Metaphysics at runtime - but it's used from clients, and so can help represent or call out issues if there are unexpected changes.

So in general, we should carefully look at updates that happen here.

Reply:
(although is this a noisy git-diff / re-ordering issue? I see `Asset` version going to `Assets` below). I guess it would be nice to verify there's no 'real' diff here.

Reply:
Ah I see your comment @anandaroop  - and nice thinking on making a live introspection query and diff'ing those, vs. this somewhat-artificial generated file that seems to be subject to a lot of noise (which isn't even used by MP itself).

That totally gives me more confidence the large diff here is in fact just generative noise + git-diffing.

Reply:
The diff you flagged makes me realize I didn't check _mutations_ though, lemme do that for good measure too…

Reply:
With mutations, a lot more noise in the diff due to alphabetization, but with the help of https://novicelab.org/jsonabc/ — those also appear to be the same ☑️